tdrs-frontend/cypress/e2e/accounts/accounts.feature

```diff
@@ -1,5 +1,41 @@
 Feature: Users can create and manage their accounts
     Scenario: A user can log in
-        When I visit the home page
-        And I log in as a new user
-        Then I see a Request Access form
\ No newline at end of file
+        When The user visits the home page
+        And The new user logs in
+        Then I see a Request Access form
+    Scenario: A new user is put in the pending state
+        Given The admin logs in
+        And The user is in begin state
+        When The user visits the home page
+        And The user logs in
```

Comment:
these are pretty well written steps, you've made the tests more declarative which is great
https://cucumber.io/docs/bdd/better-gherkin/

i think we could improve here a little by making the "user" at play more obvious, either by naming them as in the docs examples above, or parameterizing some of the shared steps to pass the username in (i.e., `When "new-cypress" logs in`)